packages/mobile/src/hooks/errors/useErrors.tsx

```diff
@@ -35,6 +37,8 @@ export const addError = (
     details,
   };
 
+  addLogs(newError);
```

Comment:
Although it makes sense to have any errors also get written into the log; It feels pretty strange to have an actual component there to handle non critical logs, the `onlyLogging` variant. Why not put handling of those errors in the `onError` handler inside of the useQuery? That way we are sure we got each occurrence, and don't miss out on any bad usage of the hook;

Perhaps add we can even add it into the `onError` on the `queryClient` itself so it covers every scenario